Samples/Desktop/D3D12HelloWorld/src/HelloConstBuffers/D3D12HelloConstBuffers.h

```diff
@@ -44,6 +44,7 @@ class D3D12HelloConstBuffers : public DXSample
     struct SceneConstantBuffer
     {
         XMFLOAT4 offset;
+        float padding[60]; // Padding so the constant buffer is 256-byte aligned.
     };
 
```

Comment:
For these structs that must be aligned, you should consider using:

```
static_assert((sizeof(SceneConstantBuffer) % 256) == 0, "CB must be 256-byte aligned");
```

Reply:
> Make sure you pull the fix to d3dx12.h into this version... See [this PR](https://github.com/microsoft/DirectX-Graphics-Samples/pull/611)

@walbourn Thanks Chuck! I though I had it and I doubled checked it was there: CD3DX12_PIPELINE_STATE_STREAM_SUBOBJECT& operator=(InnerStructType const& i) { _Type = Type; _Inner = i; return *this; }

Did you happen to see a case where that fix was missing?

Reply:
I like that, probably the same for the HLSL side too.